src/test/java/org/jabref/logic/exporter/BibtexDatabaseWriterTest.java

```diff
@@ -485,12 +485,13 @@ void writeProtectedFlag() throws Exception {
     void writeFileDirectories() throws Exception {
         metaData.setDefaultFileDirectory("\\Literature\\");
         metaData.setUserFileDirectory("defaultOwner-user", "D:\\Documents");
+        metaData.setLaTexFileDirectory("defaultOwner-user", "D:\\Documents");
 
         databaseWriter.savePartOfDatabase(bibtexContext, Collections.emptyList());
 
         assertEquals(OS.NEWLINE + "@Comment{jabref-meta: fileDirectory:\\\\Literature\\\\;}" + OS.NEWLINE +
                 OS.NEWLINE + "@Comment{jabref-meta: fileDirectory-defaultOwner-user:D:\\\\Documents;}"
-                + OS.NEWLINE, stringWriter.toString());
+                     + OS.NEWLINE + OS.NEWLINE + "@Comment{jabref-meta: fileDirectoryldefaultOwner-user:D:\\\\Documents;}" + OS.NEWLINE, stringWriter.toString());
```

Comment:
Just nit-picking here, but please use a different test path (e.g. `D:\\Latex`). Otherwise the test would pass even through you are serializing the wrong path.

Same remark applies to the other tests below too.